types/ckeditor__ckeditor5-comments/OTHER_FILES.txt

```diff
@@ -0,0 +1,12 @@
+src/annotations/editorannotations.d.ts
```

Comment:
Can you consider adding simple import tests to force including those files in the release?
this is usuallly used for types that cannot be easily tested, e.g. for plugin types, etc.
Otherwise that's easy to miss correctness of those types once released into the wild
what do you think?